<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AniEditorWindowClass</class>
 <widget class="QMainWindow" name="AniEditorWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GANI Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidgetMain">
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionSaveAll"/>
   <addaction name="separator"/>
   <addaction name="actionSetWorkingDirectory"/>
   <addaction name="separator"/>
   <addaction name="actionBackgroundColor"/>
   <addaction name="separator"/>
   <addaction name="actionCloseAll"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../Resources/MainWindow.qrc">
     <normaloff>:/MainWindow/icons/tinycolor/icons8-document-16-2.png</normaloff>:/MainWindow/icons/tinycolor/icons8-document-16-2.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../Resources/MainWindow.qrc">
     <normaloff>:/MainWindow/icons/tinycolor/icons8-folder-16.png</normaloff>:/MainWindow/icons/tinycolor/icons8-folder-16.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../Resources/MainWindow.qrc">
     <normaloff>:/MainWindow/icons/tinycolor/icons8-save-16.png</normaloff>:/MainWindow/icons/tinycolor/icons8-save-16.png</iconset>
   </property>
   <property name="text">
    <string>actionSave</string>
   </property>
   <property name="toolTip">
    <string>Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="../../Resources/MainWindow.qrc">
     <normaloff>:/MainWindow/icons/tinycolor/icons8-save-as-16.png</normaloff>:/MainWindow/icons/tinycolor/icons8-save-as-16.png</iconset>
   </property>
   <property name="text">
    <string>actionSaveAs</string>
   </property>
   <property name="toolTip">
    <string>Save File As</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="icon">
    <iconset resource="../../Resources/MainWindow.qrc">
     <normaloff>:/MainWindow/icons/tinycolor/icons8-close-16.png</normaloff>:/MainWindow/icons/tinycolor/icons8-close-16.png</iconset>
   </property>
   <property name="text">
    <string>Close All</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSetWorkingDirectory">
   <property name="icon">
    <iconset resource="../../Resources/MainWindow.qrc">
     <normaloff>:/MainWindow/icons/tinycolor/icons8-pictures-folder-16.png</normaloff>:/MainWindow/icons/tinycolor/icons8-pictures-folder-16.png</iconset>
   </property>
   <property name="text">
    <string>Set working directory</string>
   </property>
   <property name="toolTip">
    <string>Set Working Directory</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="icon">
    <iconset resource="../../Resources/MainWindow.qrc">
     <normaloff>:/MainWindow/icons/tinycolor/icons8-save-all-16 .png</normaloff>:/MainWindow/icons/tinycolor/icons8-save-all-16 .png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="toolTip">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionBackgroundColor">
   <property name="icon">
    <iconset resource="../../Resources/MainWindow.qrc">
     <normaloff>:/MainWindow/icons/tinycolor/icons8-color-16.png</normaloff>:/MainWindow/icons/tinycolor/icons8-color-16.png</iconset>
   </property>
   <property name="text">
    <string>Background Color</string>
   </property>
   <property name="toolTip">
    <string>Set Background Color</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSetTheme">
   <property name="icon">
    <iconset resource="../../Resources/MainWindow.qrc">
     <normaloff>:/MainWindow/icons/tinycolor/icons8-theme-16.png</normaloff>:/MainWindow/icons/tinycolor/icons8-theme-16.png</iconset>
   </property>
   <property name="text">
    <string>Set Theme</string>
   </property>
   <property name="iconText">
    <string>Set Theme</string>
   </property>
   <property name="toolTip">
    <string>Set Theme</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../Resources/MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
